### PR TITLE
Add tooltip before user creation

### PR DESCRIPTION
**Pull request by ndufreche**
screenshot from 2015-05-28 15 40 21

### CODE REVIEW

File: src/tb/apps/user/controllers/user.controller.js [hunk 1]
```diff
@@ -126,6 +126,9 @@ define(
                             popin.popinManager.destroy(view.popin);
                             self.indexService(require, popin);
                             Notify.success(trans('user_save_success'));
+                            if ('new' === action) {
+                                setTimeout(Notify.warning, 1500, trans('to_assign_a_user_rights_remember_to_drag_and_drop_the_user_into_a_group'));
```
Comment (hbaptiste) [inline src/tb/apps/user/controllers/user.controller.js]: You are kidding, right?

Comment (mickaelandrieu) [inline src/tb/apps/user/controllers/user.controller.js]: lol

Comment (fkroockmann) [inline src/tb/apps/user/controllers/user.controller.js]: i'm ok with that but 1500 its too much no ?

Comment (mickaelandrieu) [inline src/tb/apps/user/controllers/user.controller.js]: it's in ms, 1,5s is not too much imo

**Pull request merged**